hoomd/BoxDim.h

```diff
@@ -496,6 +496,139 @@ struct
         w.z = v.z;
         }
 
+    //! Wrap vectors back into the box
+    /*! \param pos Vector to wrap, updated to the minimum image obeying the periodic settings
+        \param vel Vector to wrap according to deformation rates, obeying the periodic settings
+        \param img Image of the vector, updated to reflect the new image
+        \param flags Vector of flags to force wrapping along certain directions
+        \post \a img and \a v are updated appropriately
+        \note \a v must not extend more than 1 image beyond the box
+    */
+   
+    HOSTDEVICE void wrap_pos_vel(Scalar3& pos, Scalar3& vel, int3& img, char3 flags = make_char3(0, 0, 0)) const
```

Comment:
This is an overloaded function, so you can use the same name but a different call signature
```suggestion
    HOSTDEVICE void wrap(Scalar3& pos, Scalar3& vel, int3& img, char3 flags = make_char3(0, 0, 0)) const
```